config.go

```diff
@@ -0,0 +1,56 @@
+package logscraper
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+var parsersByName = map[string]Parser{
+	"go":        GoLogParser,
+	"albion":    AlbionLogParser,
+	"router":    RouterLogParser,
+	"java":      JavaLogParser,
+	"yellowfin": YellowfinLogParser,
+}
+
+type SourceConfig struct {
+	Name     string `json:"name"`
+	Filename string `json:"filename"`
+	Parser   string `json:"parser"`
+}
+
+type LogScraperConfig struct {
+	Sources []SourceConfig `json:sources`
+}
+
+func LoadLogScraperConfig(file string) (*LogScraperConfig, error) {
+	if file == "" {
+		return nil, fmt.Errorf("Config file cannot be nil")
+	}
+
+	var cfg LogScraperConfig
+	bytes, err := ioutil.ReadFile(file)
+	if err == nil {
+		err = json.Unmarshal(bytes, &cfg)
+	}
+	return &cfg, err
+}
+
+func (config *LogScraperConfig) LogSources() ([]*LogSource, []error) {
+	if config.Sources == nil || len(config.Sources) == 0 {
```

Comment:
This doesn't really make sense - if you don't have config something is wrong and we should have aborted execution already or cause an error to  propagated to the caller of this function.